### ISSUE TITLE
The Bugs in the 2 databases and other ideas

### ISSUE THREAD

**Issue by Csumidex** [JazzfanRS/Radiosure-station-database#8]
First about RBI ( Radio Browser Info ) and later about RS ( Radio Sure ) databases

RBI ( Radio Browser Info )  stations - 2022 - 04 - 28  appearing on site and starting to take a look and analize the data

- What is the need for 6x URL colums if 4 of them have NO streams only minus ( - ) signs ?
NO need for all of them , can be deleted with NO streams in it

- What is the need having 2 colums with streams but the colums ar identical ? 
you do not need the colums  URL x URL y if has no streams in it 

- Why are the COUNTRY and LANGUAGE colums identical ?

- Why have NO minus ( - ) signs filled in empty rows in COUNTRY and LANGUAGE colums ?

- Take a look for the shoot of last update .... stations - 2022 - 05 - 08

https://prnt.sc/A126Adp00lXM  ..........  https://prnt.sc/VwwHApmOX4vh ..... there are hundreds of them , will have troubles in channel sorting

- Starting a huge clean up
Initially was 31.809 stations/rows . After a first clean up remain 28.682 station , and doing only half of the work . And the check of the redirects and other inapropiate signs in streams still to come .
The difference ( about 3.200 station duplicate )  moved in a aux folder for further analizis .

The recorders are 1 station 64x duplicate , another wit 24x duplicate stream .... etc.

https://prnt.sc/IlmJtruXpaCu
https://prnt.sc/_mkEJc9jQfIm

So my advice is do not merge the RBI and RS databases before the final clean up . Maybe you get a bigger database but bigger the later mass will be . 

This is for today , later will story about RS .db and other findings

**Comment by JazzfanRS** [comment]
Hello, welcome!

4-28 was an early attempt before their were other collaberators. They have made huge strides!
They have coded a conversion to the plain text that RS requires. And had just recently scripted an addition to replace RS autoupdate. @GHbasicuser and @francois-neosurf, are the IT guys, I'm just the idea guy.

**Comment by JazzfanRS** [comment]
@Csumidex 4-28 located within issues is old. it was a test of the conversion before 'the specific field's were fine tuned. The data in the separate fields is the way it was entered at RB.info, other than placing a generic' term where fields were blank. Nothing has been scripted yet. Do you even use RadioSure application? The priority was that streams worked in the conversion. And strange characters might be another language as submitted at RB.info. If you can assist with how to homogenize 'genre' in a script consult @francois-neosurf and @GHbasicuser

**Comment by Csumidex** [comment]
Using Radio Sure for 8 years or so and made a clean .db for my use , have the skill an tools for this , will tell  you how many duplicates I deleted there . Changed some skins to my prefference too . The orriginal ones were to small

**Comment by JazzfanRS** [comment]
> Further findigs do you want to continue here in this Issue topic or open an other ?

This does not need to be moved to a new issue. These items are already covered in other open issues or have already been resolved, and closed. A closed issue is used to discuss further improvements without creating new issues.

**Comment by Csumidex** [comment]
There is room for improvement. Can you rebuild RS with improvements?
@JazzfanRS
Sent me a mail to discuss it in privat

**Comment by Csumidex** [comment]
> There is room for improvement. Can you rebuild RS with improvements? @JazzfanRS Sent me a mail to discuss it in privat
